### PR TITLE
Implementing feedback on README 

### PR DESCRIPTION
**Pull request by bedroesb**
This PR implements the feedback for the joint editorial meeting on 7/03/2023. Feel free to chip in!

### CODE REVIEW

**Comment by bedroesb** [comment]
@elixir-europe/rdmkit-editors and @elixir-europe/fair-cookbook-editors Reminder!

File: README.md [hunk 1]
```diff
@@ -7,30 +7,30 @@ Both websites have their content managed and deployed using GitHub:
...
+Changes made to the [`faircookbook_rdmkit_mapping.yml`](hfaircookbook_rdmkit_mapping.yml) file will be merged in the `main` branch using pull requests and only when at least one member of each editorial team has approved the changes. The `main` branch is for this reason protected against direct changes to prevent unexpected behaviour on one of the resources.
 
 ### Mapping rules
-- Data life cycle, your role, and national resources pages are not linked to recipes.
-- Domain pages should only link to domain specific recipes.
-- Task pages should only link to recipes for generic tasks.
+- [Data life cycle](https://rdmkit.elixir-europe.org/data_life_cycle), [your role](https://rdmkit.elixir-europe.org/your_role), and [national resources](https://rdmkit.elixir-europe.org/national_resources) pages in RDMkit are not linked to FAIRCookbook recipes.
+- [Domain](https://rdmkit.elixir-europe.org/your_domain) pages in RDMkit should only link to domain specific FAIRCookbook recipes.
```
Comment (rabuono) [inline README.md:24]:
```suggestion
- [Domain](https://rdmkit.elixir-europe.org/your_domain) pages in RDMkit should only link to domain specific FAIR Cookbook recipes.
```

File: README.md [hunk 2]
```diff
@@ -7,30 +7,30 @@ Both websites have their content managed and deployed using GitHub:
...
+- [Task](https://rdmkit.elixir-europe.org/your_tasks) pages in RDMkit should only link to FAIRCookbook recipes for generic tasks.
```
Comment (rabuono) [inline README.md:25]:
```suggestion
- [Task](https://rdmkit.elixir-europe.org/your_tasks) pages in RDMkit should only link to FAIR Cookbook recipes for generic tasks.
```

File: README.md [hunk 3]
```diff
@@ -7,30 +7,30 @@ Both websites have their content managed and deployed using GitHub:
...
 
-Because new content gets created over time in both resources, we created two automations to increase the sustainability.
+Because new content gets created over time in both resources, we created two automations in the repository to increase sustainability.
 
-### Automatic updating of the titles in the mapping file
+### Automatic updating of the titles in the [mapping file](faircookbook_rdmkit_mapping.yml)
 
-Titles can change, and since the resources list links towards each other including the title, we've put a system in place to prevent they get out of sync with the actual titles. A GitHub action will weekly check for title changes using information from the sidebars of each resource and create a pull request updating the titles in the main YAML file accordingly.
+Titles can change, and since the resources list links towards each other including the title, we've put a system in place to prevent they get out of sync with the actual titles. A GitHub Action will weekly check for title changes using information from the sidebars of RDMkit and FAIRCookbook and create a pull request updating the titles in the main YAML file accordingly.
```
Comment (rabuono) [inline README.md:33]:
```suggestion
Over time, page titles can change in both resources. We have thus put in place a system to keep displayed titles of linked pages in sync with the original resource. A GitHub Action will weekly check for title changes using information from the sidebars of RDMkit and FAIR Cookbook and create a pull request updating the titles in the main YAML file where needed.
```

**Review by rabuono — APPROVED** [review]
Thanks!

**Review by daniwelter — APPROVED** [review]
Looks good to me

**Comment by bedroesb** [comment]
wohoow! Should we merge or wait till the next meeting?

**Pull request merged**